README.md

```diff
@@ -67,16 +68,28 @@ parser.parse().then((items) => {
   */
 })
 
-// 3. You can pass the spreadsheet ID when call parse method.
+// 3. You can pass the spreadsheet ID when call parse method:
 parser.parse(spreadsheetId).then((items) => {
   // items should be [{"a":1,"b":2,"c":3},{"a":4,"b":5,"c":6},{"a":7,"b":8,"c":9}]
 })
 
 
-// 4. You can also pass the name of specific sheet to get.
+// 4. You can also retrieve a specific sheet to get by either passing the sheet name as a string (since v1.1.0):
 parser.parse(spreadsheetId, 'Sheet2').then((items) => {
   // items should be [{"a":10,"b":20,"c":30},{"a":40,"b":50,"c":60},{"a":70,"b":80,"c":90}]
 })
+// ...or as an object (since v1.3.0) that specifies the sheet's name or ID. If both are provided, sheet ID is used:
+parser.parse(spreadsheetId, { sheetId: '123456789' }).then((items) => {
```

Comment:
Can you change this sheetId to real one? 784337977 is real id of real example sheet.